### DISCUSSION TITLE
Makefile:27: compile_frontend

### DISCUSSION THREAD

**Discussion by dd123-a** [ricklamers/gpt-code-ui#55] [Q&A]
C:\Users\DrTang\Desktop\gpt-code-ui>make compile_frontend
cd frontend && \
npm install && \
npm run build && \
find ../gpt_code_ui/webapp/static -mindepth 1 ! -name '.gitignore' -delete && \
rsync -av dist/ ../gpt_code_ui/webapp/static

up to date, audited 255 packages in 4m

66 packages are looking for funding
  run `npm fund` for details

2 moderate severity vulnerabilities

To address all issues, run:
  npm audit fix

Run `npm audit` for details.

> gpt-code-frontend@0.0.0 build
> echo Building version $VITE_APP_VERSION && tsc && vite build

Building version $VITE_APP_VERSION
vite v4.3.9 building for production...
✓ 1606 modules transformed.
dist/index.html                     0.48 kB │ gzip:   0.31 kB
dist/assets/index-18b36754.css      2.75 kB │ gzip:   1.12 kB
dist/assets/index-9bf1a79d.js   1,092.83 kB │ gzip: 365.48 kB

(!) Some chunks are larger than 500 kBs after minification. Consider:
- Using dynamic import() to code-split the application
- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
✓ built in 4.29s
FIND: 参数格式不正确
make: *** [Makefile:27: compile_frontend] Error 2

**Comment by ricklamers** [comment]
Duplicate of https://github.com/ricklamers/gpt-code-ui/issues/54

Please use GNU grep
